core/platform/auth/firebase_auth_services_test.py

```diff
@@ -453,8 +453,8 @@ def assert_is_not_super_admin(self, uid):
     def assert_is_disabled(self, uid):
         """Asserts that the given ID is a disabled account.
 
-        NOTE: This method can only be called after the instance has been
-        installed to a test case!
+        NOTE: This method can only be called after the stub has been installed
```

Comment:
Ah looks better. Maybe update the above docstrings too!

Reply:
Done.